app/build.gradle

```diff
@@ -118,6 +112,13 @@ dependencies {
     //gson
     implementation 'com.google.code.gson:gson:2.10.1'
 
+    // koin
+    def koinVersion = "3.4.3"
```

Comment:
Suggestion: I would move it into project `build.gradle`. We already have one defined version here.